lib/normalizers/v1.js

```diff
@@ -527,7 +527,7 @@ module.exports = {
         // else
         // if id is falsy, replace the id
         // if retainIds is false, replace the id
-        !((options.retainIds && collection.id) || options.noDefaults) && (collection.id = util.uid());
```

Comment:
Why is collection.id not updated when `retainIds` is set to false?

Reply:
Addressed